new-packages/ts-project/__tests__/utils.ts

```diff
@@ -0,0 +1,114 @@
+import realFs from 'fs';
+import * as fs from 'fs-extra';
+import outdent from 'outdent';
+import path from 'path';
+import { onExit } from 'signal-exit';
+import { temporaryDirectory } from 'tempy';
+import typescript from 'typescript';
+import { createProject } from '../src/index';
+
+export const js = outdent;
+export const ts = outdent;
+export const tsx = outdent;
```

Comment:
I will create a follow-up PR that will introduce auto-formatting of the code "rendered" using those utils